tools/src/main/resources/ui/html/raw.html

```diff
@@ -149,31 +149,27 @@ <h1 class="dash-title">Full Report</h1>
                 </div>
                 <div class="spur-card-title">Raw Data of all Applications</div>
               </div>
-              <div class="card-body ">
+              <div class="card-body">
                 <div id="all-apps-raw-data">
-                  <table id="all-apps-raw-data-table" class="table data-table display datatable" style="width:100%">
+                  <table id="all-apps-raw-data-table" class="table data-table display wrap" style="width:100%">
                     <thead>
                     <tr>
                       <th>App Name</th>
                       <th>App ID</th>
-                      <th><span data-toggle="tooltip" data-placement="top"
-                                title="Time duration that includes only SQL/Dataframe queries">
-                                        SQL DF Duration</span></th>
-                      <th><span data-toggle="tooltip" data-placement="top"
-                                title="Amount of time spent in tasks of SQL Dataframe operations measured in Wall-Clock">
-                                        SQL Dataframe Task Duration</span></th>
-                      <th><span data-toggle="tooltip" data-placement="top"
-                                title="Highlighted in blue are apps that were missing application finished event from the log">
-                                        App Duration</span></th>
-                      <th><span data-toggle="tooltip" data-placement="top"
-                                title="This is an estimate at how much time the tasks spent doing processing on the CPU vs waiting on IO. Shaded red when it is below 40%">
-                                        Executor CPU Time Percent</span></th>
+                      <th>User</th>
+                      <th>Start Time</th>
+                      <th>Recommendation</th>
+                      <th>Total Speedup Factor</th>
+                      <th>App Duration</th>
                       <th>Estimated GPU Duration</th>
+                      <th>GPU Time Saved</th>
+                      <th>SQL DF Duration</th>
+                      <th>GPU Opportunity</th>
                       <th>Unsupported Task Duration</th>
-                      <th>Speed-up Duration</th>
```

Comment:
missing Supported SQL df task duration, potential problems, app duration estimated, task speedup factor

Reply:
Added the remaining fields